fix: jobs page (#485)
ory/web: src/markdown/jobs/marketing-campaign-manager.mdx

metaDescription: 'Read the Marketing Campaign Manager job description.'
lastUpdatedAt: 'December 12, 2021'
description:
  'Devise and execute marketing campaigns to expand Ory’s reach among prospects
  and customers across multiple channels.'
location: 'Munich, Germany'
position: 'Full Time'
---

import { About, WorkAtOry } from './shared'

## Marketing Campaign Manager

<About />

> As Marketing Campaign Manager, you will devise and execute marketing campaigns
> to expand Ory’s reach among prospects and customers across multiple channels.

Reviewer: ```suggestion
As Marketing Campaign Manager you will strategize, execute, and optimize data-driven demand generation campaigns on an international level to expand Ory’s reach among prospects and customers across multiple channels.
```

### What You'll Do

> - Conceptualize and execute marketing campaigns to expand Ory’s reach among
>   prospects and customers across multiple channels

Reviewer: ```suggestion
- Conceptualize, execute and optimize data-driven demand generation campaign to expand Ory’s reach among
  prospects and customers across multiple channels.
```

> - Create and manage a Digital Marketing campaign calendar

Reviewer: ```suggestion
-  Develop and deploy integrated marketing campaigns aligning with internal teams, developing audience segments, building up an E-Mail base, and planing top-funnel virtual events.
```

- Assist team workflow and communicate across different teams

> - Collaborate with external Stakeholder when necessary

Reviewer: ```suggestion
- Collaborate with internal and external Stakeholders to ensure proper messaging, timing, and execution of multi-channel campaigns
```

> - Research competitor strategies and industry standards in brand and performance

Reviewer: ```suggestion
- Apply an experimentation mindset to marketing campaigns in order to drive, test, learn and optimize. Continuously conduct A/B testing across all channels
```

>   campaigns

Reviewer: ```suggestion
```

> - Establish innovative and flexible content creation and campaign management
>   workflows

Reviewer: ```suggestion
- Establish innovative and flexible content for our audience of developers and IT leaders and build prescriptive persona-based buyer journeys as the foundation for campaign strategy
```

> - Make creative, engaging, and impactful marketing content in collaboration with
>   team members

Reviewer: ```suggestion
- Make creative, engaging, and impactful marketing content in collaboration with
  team members for our audience of developers and IT leaders
```

- Grow and shape Ory’s presence as global brand with open source roots

> - Explore new channels for reaching relevant audiences for Ory Cloud

Reviewer: ```suggestion
- Explore new channels for reaching relevant audiences for Ory Cloud
```
```suggestion
Provide ongoing reporting and campaign insights
```

### We’re looking for someone who has

- 5+ years experience in digital and performance marketing
- Strong writing skills and ability to explain technical concepts to a wide

>   audience with varying levels of technical expertise

Reviewer: ```suggestion
  audience with varying levels of technical expertise
```
```suggestion
  - Experience measure tactic and channel impacts
```
```suggestion
  - Experience working in a B2B software marketing team
```
```suggestion
Experience with A/B testing, experiment design, campaign analysis (attribution reporting is a plus)
```

Author: ????

Author: which one is it? 
merging the last one.

- Experience in creative development and production and collaboration with
  external stakeholders
- Advanced understanding of industry standards and emerging trends in digital
  marketing

> - Ability to self-manage and collaborate across teams as well as reporting

Reviewer: ```suggestion
- Strong project management, organizational, and communication skills and ability to self-manage and collaborate across teams
```

> - Generating marketing insights and willingness to try out creative new ideas
>   and iterate on them

Reviewer: ```suggestion
- Willingness and flexibility to work both independently as well as part of a team
```
